$NetBSD: patch-build_moz.configure_rust.configure,v 1.3 2020/06/07 13:06:00 wiz Exp $

* Do not match rumprun toolchain for NetBSD,
  narrowed should be one not two.

--- build/moz.configure/rust.configure.orig	2020-05-28 00:01:39.000000000 +0000
+++ build/moz.configure/rust.configure
@@ -330,7 +330,8 @@ def rust_triple_alias(host_or_target):
             narrowed = [
                 c for c in candidates
                 if c.target.raw_os == host_or_target.raw_os and
-                   c.target.raw_cpu == host_or_target.raw_cpu
+                   c.target.raw_cpu == host_or_target.raw_cpu and
+                   not c.rust_target.endswith("-rumprun-netbsd")
             ]
             if len(narrowed) == 1:
                 return narrowed[0].rust_target
